doc/changes/latest.inc

```diff
@@ -133,6 +133,8 @@ Changelog
 
 - Expose the number of ICA iterations during the fitting procedure via the ``n_iter_`` attribute of :class:`mne.preprocessing.ICA` by `Richard Höchenberger`_
 
+- :func:`mne.grand_average` now produces a warning when only a single dataset was passed, instead of raising by `Richard Höchenberger`_
```

Comment:
```suggestion
- :func:`mne.grand_average` now produces a warning when only a single dataset was passed, instead of raising an error by `Richard Höchenberger`_
```